### DISCUSSION TITLE
Documentation on how to deploy keycloak-operator on Kubernetes

### DISCUSSION THREAD

**Discussion by droslean** [keycloak/keycloak#14477] [Archive]
Is there any documentation that describes how to deploy the new keycloak-operator in kubernetes?

**Comment by droslean** [comment]
I tried https://artifacthub.io/packages/olm/community-operators/keycloak-operator but it is installing version `19.0.2` instead of `20.0.0-alpha.6`

**Comment by dolfim-ibm** [comment]
I had a similar problem, then I found the actual documentation here:
- https://www.keycloak.org/operator/installation
- https://www.keycloak.org/operator/basic-deployment

Once you reach the Keycloak guides at https://www.keycloak.org/guides#operator, the Operator section (a bit at the bottom) is all about the new operator.

**Reply by droslean** [reply]
Thanks @dolfim-ibm, but this is only for OLM on Openshift. OLM in Kubernetes still uses the old operator, and also the documentation you shared is for the old operator as well.

**Comment by sschu** [comment]
https://github.com/keycloak/keycloak/blob/main/operator/README.md gives at least some hints.
